src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,124 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { SearchLink } from '../SearchLink';
+import { Sex } from '../../types';
+import { CENTURIES_TO_FILTER, DEFAULT_PARAMS } from '../../utils/constants';
+import { getSearchWith } from '../../utils/searchHelper';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchPrams] = useSearchParams();
+  const sexFromSearch = searchParams.get('sex') || '';
+  const centuriesFromSearch = searchParams.getAll('centuries') || [];
+  const queryFromSearch = searchParams.get('query') || '';
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const query = event.target.value || null;
+
+    setSearchPrams(
+      getSearchWith(searchParams, { query }),
+    );
+  };
+
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        <SearchLink
+          className={classNames({ 'is-active': !sexFromSearch })}
+          params={{ sex: null }}
+        >
+          All
+        </SearchLink>
+
+        {Object.entries(Sex).map(sex => (
+          <SearchLink
+            className={classNames({ 'is-active': sexFromSearch === sex[1] })}
+            params={{ sex: sex[1] }}
+          >
+            {sex[0]}
+          </SearchLink>
+        ))}
+      </p>
+
+      <div className="panel-block">
+        <p className="control has-icons-left">
+          <input
```

Comment:
![image](https://github.com/mate-academy/react_people-table-advanced/assets/139622699/e4ac6100-9f59-4fcd-a22a-84776f3e4f7b)
check this pls

Reply:
Same is for working version, but good point, ill work it out.